build.gradle

```diff
@@ -820,6 +821,18 @@ public abstract class RunHotswappableMinecraftTask extends RunMinecraftTask {
         }
         this.classpath(project.java17DependenciesCfg)
     }
+
+    public void setup(Project project) {
+        super.setup(project)
+        if (project.usesMixins.toBoolean()) {
+            this.extraJvmArgs.addAll(project.provider(() -> {
+                def mixinCfg = project.configurations.detachedConfiguration(project.dependencies.create(project.mixinProviderSpec))
+                mixinCfg.canBeConsumed = false
+                mixinCfg.transitive = false
+                enableHotswap ? ["-agent:" + mixinCfg.singleFile.absolutePath] : []
```

Comment:
Shouldn't this be `-javaagent:`? See (from the output of `java`):
```
    -javaagent:<jarpath>[=<options>]
                  load Java programming language agent, see java.lang.instrument
```
By the way, you can tell if the agent is working if you see messages coming from a logger called `mixin.agent` in the log (they should appear even if you're not hot swapping anything).